wiki/Contests/PHFC/2024/en.md

```diff
@@ -0,0 +1,67 @@
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 35% of the prize pool |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 15% of the prize pool |
+
+## Organisation
+
+The osu! Philippines Mascots Fanart Contest 2024 is run by the following staff:
+
+| Position | Member(s) |
+| :-- | :-- |
+| Host | ::{ flag=NL }:: [Albionthegreat](https://osu.ppy.sh/users/9853595), ::{ flag=PH }:: [SurfChu85](https://osu.ppy.sh/users/4469895), ::{ flag=PH }:: [topecnz](https://osu.ppy.sh/users/2103927) |
+| Contest moderator | ::{ flag=NL }:: [Albionthegreat](https://osu.ppy.sh/users/9853595), ::{ flag=CN }:: [Sakura006](https://osu.ppy.sh/users/10365024), ::{ flag=PH }::  [topecnz](https://osu.ppy.sh/users/2103927) |
+| Developer | ::{ flag=PH }:: [-Ichinose](https://osu.ppy.sh/users/12635292), ::{ flag=PH }:: [topecnz](https://osu.ppy.sh/users/2103927) |
+| Statistician | ::{ flag=NL }:: [Albionthegreat](https://osu.ppy.sh/users/9853595), ::{ flag=PH }:: [SurfChu85](https://osu.ppy.sh/users/4469895) |
+| Wiki editor | ::{ flag=PH }:: [SurfChu85](https://osu.ppy.sh/users/4469895) |
+
+## Links
+
+- Submission form
+- Discussion thread
+- [osu! Philippines Discord server](https://discord.gg/officialosuph)
+
+## Rulesets
+
+- By submitting your artwork to this contest, you give osu! Philppines the permission to use it for any promotional material for its activities, not limited to events, promotions, and the like. You will retain all of the rights to your work and you are permitted to sell or distribute it.
+- Participants must have the Philippine flag on their profiles.
+- All entries MUST follow the contest theme and contain AT LEAST two of osu! Philippines' community mascots, one of which must be Primrea. You may look at these links for your reference:
+  - Primrea, the osu! Philippines/`#filipino` mascot
+  - Primreadon, the osu!taiko PH mascot
+  - Punica, the osu!catch PH mascot
+  - [Krysta, the osu!mania PH mascot](https://up.topecnz.net/maniaph_krysta_presskitv5.pdf)
+- All entries MUST be suitable for viewing by all ages.
+- Your submission must be your own original work. Redrawing and composite works are not allowed.
+- You can only submit ONE entry. You may reupload your entry for adjustments as many times as you want, but once the submission deadline has passed, only your latest submission will be considered.
+- Generative art programs or methods (like AI) are not permitted.
+- Participants must keep a full file containing all of their entry's layers available. The staff will only ask you for this if they think it is needed to ascertain any claims regarding plagiarism or the use of generative art tools. If you do not/cannot provide this within a timely manner when asked, your entry will be disqualified.
+- Artist attribution (names, social media handles, etc.) must be limited to no more than three lines total and confined to any corner of your entry.
```

Comment:
```suggestion
- Artist attribution (names, social media handles, etc.) must be limited to no more than three lines total and be confined in a corner.
```

Reply:
this is likely copied from the wording in official art contests, but if removing "you" is the goal here then this would sound ok enough

```suggestion
- Artist attribution (names, social media handles, etc.) must be limited to no more than three lines total and be confined to any corner of the entry.
```